### PR TITLE
Feature enter site/select specific date

### LINKED ISSUE #3: Create Initial layout
**Describe the solution you'd like**
As a user, I should be able to see the layout of the dashboard clearly, so that I can understand the navigation of the site.

**Additional context**
Should be visible:
[ ] Header (Title)
[ ] Body (POD, see other issue)
[ ] Footer (About, FAQ, Contact Us)

### LINKED ISSUE #7: Go Home
**Describe the solution you'd like**
As a user, I should be able to click a home link or app title in the header, so that I can go back to the home POD page.

**Additional context**
Be able to navigate back home when on a page that is not the home page

### LINKED ISSUE #8: Select a date to view that POD
**Describe the solution you'd like**
As a user, I should be able to make a date input, so that I can view the Photo of the Day for the date selected.

### LINKED ISSUE #9: Connect Date Input to Fetch call
Take input date and connect it to the fetch call url (see example in notion for how it is formatted) which will make the api call for that date

### PR DESCRIPTION
**Pull request by lbmerchant93**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Create entrance page, click route to be taken into the site and greeted with the POTD.
Move Home route to Header component.
Make Title of site a link to Home.
Add styling to POTD component.
Conditional render for Copyright credits.
Fetch Photo by specific date.
Limit calendar input to today's date.

## *Instructions for reviewer:*
*Where should the reviewer start?*
Header line 3
POTD line 17

*How should this be manually tested?*
Selection on calendar should work. Should be greeted with welcome screen and routes work properly.

## *Issues:*
Look into specific routes by date once through the MVP
Make welcome page better (Add some details like "Welcome to the Vueniverse, Click here to enter the sight", take up more space, add more fun emojis?)

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -92,8 +91,17 @@ footer {
...
+  flex-direction: column;
+  justify-content: center;
+  animation: fadeIn ease 2s;
 }
 
+@keyframes fadeIn {
+0% {opacity:0;}
+100% {opacity:1;}
+} */
+
```
Comment (colefiscus) [inline src/App.vue:104]: Nice! I gotta steal this.

File: src/views/PicOfTheDay.vue [hunk 2]
```diff
@@ -11,11 +14,19 @@
...
+      :src="potd.url" 
+      class="photo" 
+      v-if="potd.media_type === 'image'" :alt="potd.title"
+    >
+    <section class="information">
+     <h2 class="potd-date">{{ potd.date }}</h2>
+     <h3 class="potd-title">{{ potd.title }}</h3>
+     <h4 class="potd-copyright" v-if="potd.copyright">{{ potd.copyright }}</h4>
+    <p class="potd-explanation">{{ potd.explanation }}</p>
+    </section>
```
Comment (colefiscus) [inline src/views/PicOfTheDay.vue:27]: I'm just being picky but I can't tell if these are indented.

**Review by colefiscus — APPROVED** [review]
Looks great! I really can't find much to comment on.

**Review by richardltyler — APPROVED** [review]
Looks good!

**Pull request merged**